### PR TITLE
Add support for values that include an equals

### PR DESCRIPTION
**Pull request by drapergeek**

### CODE REVIEW

File: lib/envy.ex [hunk 1]
```diff
@@ -70,11 +72,18 @@ defmodule Envy do
   defp parse_line(line) do
     [key, value] = line
       |> String.strip
-      |> String.split("=")
+      |> String.split(@key_value_delimeter)
```
Comment (BlakeWilliams) [inline lib/envy.ex]: If we did `|> String.split(@key_value_delimeter, parts: 2)` would that also make the test pass?

Love the extraction of `=` to a module attribute by the way!

Comment (drapergeek) [inline lib/envy.ex]: TIL!

Updated

**Comment by BlakeWilliams** [comment]
Nice find, thanks!

**Comment by drapergeek** [comment]
Thank you!

Mind dropping a minor release?

**Comment by BlakeWilliams** [comment]
1.1.0 should be available now. 😃

**Pull request merged**